docs/concepts/labels.md

```diff
@@ -22,10 +22,11 @@ Below are the list of available feature labels that can be used to define [Volum
-| [**Failure Mode**](/docs/concepts/replication/#ondat-failure-modes) | `storageos.com/failure-mode`   | `hard`, `soft`, `alwayson`, or `threshold` integers starting from `0` to `5`                   | Sets the failure mode for a volume, either explicitly using a failure mode or implicitly using a replica threshold. The default setting of a failure mode is `hard`.                                                                           |
+| [**Failure Mode**](/docs/concepts/replication/#ondat-failure-modes) | `storageos.com/failure-mode`   | `hard`, `soft`, `alwayson`, or `threshold` integers starting from `0` to `5`                   | Sets the failure mode for a volume, either explicitly using a failure mode or implicitly using a replica threshold. The default failure mode is set to `hard`.                                                                                 |
 | [**Replication**](/docs/concepts/replication/)                      | `storageos.com/replicas`       | `integers` starting from `0` to `5`                                                            | Sets the number of replicas, for example full copies of the data across nodes. Typically `1` or `2` replicas is sufficient (`2` or `3` instances of the data). Latency implications need to be assessed when using **more than** `2` replicas. |
 | [**Topology-Aware Placement**](/docs/concepts/tap/)                 | `storageos.com/topology-aware` | `true` / `false`                                                                               | Enables or disables Ondat Topology-Aware Placement.                                                                                                                                                                                            |
 | [**Topology Domain Key**](/docs/concepts/tap/#topology-domains)     | `storageos.com/topology-key`   | custom region, read as a [string](https://en.wikipedia.org/wiki/String_%28computer_science%29) | Define the failure domain for the node by using a custom key. If you don't define a custom key, the label defaults to the `topology.kubernetes.io/zone` value.                                                                                 |
+| [**Squash Mode**](/docs/concepts/rwx)                               | `storageos.com/nfs-squash`     | `root`, `rootuid`, `all`, `none`                                                               | Define the squash mode that will be used with Ondat Files - `ReadWriteMany` (RWX) volumes. The default squash mode is set to `all`.                                                                                                            |
```

Comment:
I would make a reference to the linux file permissions as people might not be familiar with NFS squash config param. 

I don't think we need to explain to much, but I would go for sth like:

```
Define the squash mode that will be used with Ondat Files - `ReadWriteMany` (RWX) volumes to set the file ownership in the share.
```

Reply:
fixed thanks!